static/svgs/favorite/favorite_type-gray.svg

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You may want to replace this with an svg matching the love and remix icons in file format.